app/errors.js

```diff
@@ -10,5 +10,7 @@ exports.DEFAULT_ERROR = 'default_error';
 exports.defaultError = message => internalError(message, exports.DEFAULT_ERROR);
 
 exports.BAD_REQUEST_ERROR = 'bad_request';
-
 exports.badRequest = message => internalError(message, exports.BAD_REQUEST_ERROR);
+
+exports.UNAUTHORIZED_ERROR = 'unauthorized_request';
```

Comment:
This should be an authentication error, not an authorization error (by the way you use it in the controller)

Reply:
Perfect, I have changed it by request forbidden.